docs/running-on-kubernetes.md

```diff
@@ -3,23 +3,34 @@ layout: global
 title: Running Spark on Kubernetes
 ---
 
-Support for running on [Kubernetes](https://kubernetes.io/docs/whatisk8s/) is available in experimental status. The feature set is
-currently limited and not well-tested. This should not be used in production environments.
+Support for running on [Kubernetes](https://kubernetes.io/docs/whatisk8s/) is available in experimental status. The
```

Comment:
@ash211 @erikerlandson @kimoonkim @lins05 

The documentation changes are by far the most important part of this change, as well as the workflow in general. This is our chance to catch any changes in the user's workflow as described in the documentation and weigh if it is still reasonable to move forward with this overhaul.